docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -805,9 +805,15 @@ If the extension solves a general problem, it is very handy for Quarkus users to
 But this comes with some responsibility for you, keeping it up to date with Quarkus minor releases (every month or so at the moment).
 When in doubt, have a conversation with the community in the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group]. We can make a collective decision.
 
-[NOTE]
-As for now, the process to propose a new extension is not defined yet.
-Your best chance is to present your extension on the https://groups.google.com/forum/#!forum/quarkus-dev[Quarkus Google Group] and wait for an official invitation to join the Quarkus Platform.
+If you want to contribute a new extension, we strongly suggest you to host it in the Quarkiverse Hub.
+
+*Quarkiverse Hub*
+
+link:https://github.com/quarkiverse[Quarkiverse Hub] is the GitHub organization that provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+You can get started by creating an link:https://github.com/quarkusio/quarkus/issues/new/choose[Extension Request] issue (check first if one wasn't already submitted link:https://github.com/quarkusio/quarkus/labels/kind%2Fextension-proposal[here]) and asking to lead it.
+We'll take care provisioning a new repository and set it up to our tooling.
+
```

Comment:
I wonder if we should somehow say that we give the key of the repository to the requester?

Reply:
What key are you referring to? The repository owner has limited access to the repository settings AFAIK

Reply:
Well it's more than people are totally independent to manage their project and releases. I think it's an important factor when taking the decision of Quarkiverse vs my own repo.

But maybe it's not worth it. I'll let you judge of that. Feel free to merge if you don't think it's necessary.

Reply:
I see, that's a good idea and a nice motivation to use Quarkiverse. I reworded it a bit, let me know how that looks now